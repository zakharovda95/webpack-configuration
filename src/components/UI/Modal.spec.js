import { ModalComponent } from './Modal.component';

describe('Modal', () => {
  let modal;

  beforeEach(() => {
    modal = new ModalComponent();
  });

  test('should be defined', () => {
    expect(modal).toBeDefined();
  });

  test('should be instance of HeaderComponent', () => {
    expect(modal).toBeInstanceOf(ModalComponent);
  });

  test('should include all elements', () => {
    expect(modal).toMatchSnapshot();
  });
});
