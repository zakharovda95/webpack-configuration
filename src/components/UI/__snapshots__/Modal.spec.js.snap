// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal should include all elements 1`] = `
ModalComponent {
  "component": <div
    aria-hidden="true"
    aria-labelledby="exampleModalCenterTitle"
    class="modal fade"
    id="exampleModalCenter"
    role="dialog"
    tabindex="-1"
  >
    
        
    <div
      class="modal-dialog modal-dialog-centered"
      role="document"
    >
      
            
      <div
        class="modal-content"
      >
        
                
        <div
          class="modal-header"
        >
          
                    
          <h5
            class="modal-title"
            id="exampleModalLongTitle"
          >
            Новый таск
          </h5>
          
                
        </div>
        
                
        <div
          class="modal-body"
        >
          
                  
                
        </div>
        
            
      </div>
      
        
    </div>
    
    
  </div>,
}
`;
